CHANGELOG_UNRELEASED.md

```diff
@@ -1,3 +1,4 @@
 
 ### Changed:
-* Navigation UI has been redesigned with aesthetic improvements and better and more clear information about the route
\ No newline at end of file
+* Indicators of google maps are hidden to keep those of situm
```

Comment:
* Changes in GoogleMaps styles to adjust the colors and lightness of some cartographic elements. 
* Changes in GoogleMaps styles to hide GoogleMaps POI names.
* Changed the behaviour of the module when setShowTextPois is not set. Now the default value is false so only the POI icons will appear on the map.